### PR TITLE
Fix PerfCurve Normalization Dependency in HybridEvapCoolingModel

### LINKED ISSUE #7828: Remove PerfCurve Normalization in HybridEvapCoolingModel
Issue overview
--------------
While fixing issue #7576 which involves fixing a bug for min/max table normalization inside of the btwxt library, and then correctly normalizing Lookup Table curves inside CurveManager.cc, I realized that the HybridEvapCoolingModel is unnecessarily using the returned Normalization Reference to un-normalize curve values which are being normalized on input. Instead of doing this, we can now rely on the user to specify in the idf how the data should be normalized, and the HybridEvapCoolingModel will behave appropriately. This issue depends on pull request #7630 to be in place before fixing.

### Details
- Platform: All
- Version of EnergyPlus: 9.2

### Checklist
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### LINKED ISSUE #7846: HybridEvapCoolingModel Curve Value Calculations are inconsistent and inaccurate
Issue overview
--------------
The calculation of the unscaled supply air mass flow rate (UnscaledMsa) inside of HybridEvapCoolingModel.cc is inconsistent. In some areas, the UnscaledMsa is simply a ratio, or fraction of the mass mass flow rate (which would be unitless), and in others it appears to be an actual measurement of  mass flow rate. This mass flow rate is fed into the `CalculateCurveVal` function which is used to determine resource consumption for the units mode settings. This issue was found while fixing issue #7828. While debugging the integration tests for HybridEvapCoolingModel, I noticed that the inputs for calls to the `CalculateCurveVal` function did not always seem to match what was expected. This defect produces innacurate results for the simulation of resource consumption for this type of equipment. This defect can be validated by checking that the results returned from a call to `CalculateCurveVal` match the expected calculation when a proper unscaled mass flow rate is fed as an input. 

### Details
Some additional details for this issue (if relevant):
 - Platform: All
 - Version of EnergyPlus: 9.2

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by ajscimone**
Pull request overview
---------------------
 - Fixes #7828 and #7846
 - This pull request fixes the HybridEvapCoolingModel dependency on a global performance curve normalization value. The HybridEvapCoolingModel relied on this normalization value to reverse the normalization of performance curves in it's calculations. This is an unnecessary and inefficient step since the curves can simply be left un-normalized in from the input files, and proceed with calculations as normal. This pull request removes this calculation step, as well as the global storage of the normalization reference.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by Myoldmopar** [comment]
Definitely still a few changes away from this being ready.  I'll be watching to see new commits coming for this so we can get it in, along with the other hybrid PR.

**Comment by Myoldmopar** [comment]
I wouldn't necessarily sweat that convection unit test failure just yet.  I saw it locally as well, changed some code and it worked, so thought I had fixed it, but @mitchute has also seen this on his machine.

Other than that it is looking pretty clean from CI's view.  Anything remaining on your end before final review?

File: src/EnergyPlus/HybridUnitaryAirConditioners.cc [hunk 1]
```diff
@@ -504,13 +494,11 @@ namespace HybridUnitaryAirConditioners {
...
         static std::string const RoutineName("GetInputZoneHybridUnitaryAirConditioners: ");
-        MaxNumbers = 0;
-        MaxAlphas = 0;
         CurrentModuleObject = "ZoneHVAC:HybridUnitaryHVAC";
         NumZoneHybridEvap = inputProcessor->getNumObjectsFound(CurrentModuleObject);
         inputProcessor->getObjectDefMaxArgs(CurrentModuleObject, NumFields, NumAlphas, NumNumbers);
-        MaxNumbers = max(MaxNumbers, NumNumbers);
-        MaxAlphas = max(MaxAlphas, NumAlphas);
+        int MaxNumbers = max(0, NumNumbers);    // Maximum number of numeric fields in all objects
+        int MaxAlphas = max(0, NumAlphas);      // Maximum number of alpha fields in all objects
```
Comment (Myoldmopar) [inline src/EnergyPlus/HybridUnitaryAirConditioners.cc:501]: :+1:

File: src/EnergyPlus/HybridEvapCoolingModel.hh [hunk 2]
```diff
@@ -85,14 +85,12 @@ namespace HybridEvapCoolingModel {
...
-        void AddItem(Real64 X, Real64 Y, Real64 M)
+        std::vector<Real64> MassFlowRatio;
+        std::vector<Real64> OutdoorAirFraction;
+        void AddItem(Real64 X, Real64 Y)
         {
-            PointX.push_back(X);
-            PointY.push_back(Y);
-            PointMeta.push_back(M);
+            MassFlowRatio.push_back(X);
+            OutdoorAirFraction.push_back(Y);
```
Comment (Myoldmopar) [inline src/EnergyPlus/HybridEvapCoolingModel.hh:93]: Simpler.  Nice.

File: testfiles/UnitaryHybridAC_DedicatedOutsideAir.idf [hunk 3]
```diff
@@ -2215,7 +2215,7 @@
     Mode1_ESP_lookup,        !- Name
     Mode1_IndependentVariableList,  !- Independent Variable List Name
     DivisorOnly,             !- Normalization Method
-    3.0176,                  !- Normalization Divisor
+    ,                        !- Normalization Divisor
```
Comment (Myoldmopar) [inline testfiles/UnitaryHybridAC_DedicatedOutsideAir.idf:2218]: Fine with me.

**Review by Myoldmopar — APPROVED** [review]
This improved normalization works for me.

**Comment by Myoldmopar** [comment]
CI is happy here.  The changes are only affecting the hybrid code.   Things seem simpler now.  Merging.

**Comment by ajscimone** [comment]
@Myoldmopar Thanks Edwin!

**Pull request merged**